config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml

```diff
@@ -104,6 +104,84 @@ presubmits:
         args:
         - runner.sh
         - ./scripts/ci-test.sh
+        env:
+        - name: KUBEBUILDER_ENVTEST_KUBERNETES_VERSION
+          value: "1.22.0"
+        resources:
+          requests:
+            cpu: 7300m
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api
+      testgrid-tab-name: capi-pr-test-main
```

Comment:
If my assumptions about how testgrid dashboards are correct, this would mean the job results are all in the same dashboard. Do we want to split that up to have one dashboard per job? (Imho this makes it easier to identify Kubernetes-version specific job failures (especially flakes))

Reply:
*ping* @vincepri  :)

Reply:
Ah according to the failed test we *have* to change the tab name because duplicate names are not supported